JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -215,7 +222,7 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
         lCI  = fit[["ci.lb_adj"]][mean_pos,1],
```

Comment:
Is the dot there for column folding? If not please just use camelCase

Reply:
no, the dot is there because that's how the `weightr` package names the object